airflow/www/app.py

```diff
@@ -60,6 +61,11 @@ def create_app(config=None, session=None, testing=False, app_name="Airflow"):
         )
     app.secret_key = conf.get('webserver', 'SECRET_KEY')
 
+    session_lifetime_days = 30
+    if conf.has_option('webserver', 'SESSION_LIFETIME_DAYS'):
+        session_lifetime_days = conf.getint('webserver', 'SESSION_LIFETIME_DAYS')
```

Comment:
This can be changed to :

```python
session_lifetime_days = conf.getint('webserver', 'SESSION_LIFETIME_DAYS', fallback=30)
```

Reply:
We should also add this config to https://github.com/apache/airflow/blob/master/airflow/config_templates/config.yml and https://github.com/apache/airflow/blob/master/airflow/config_templates/default_airflow.cfg (If you are using pre-commit hooks `default_airflow.cfg` would be auto-generated for you from `config.yml` when you try to commit

Reply:
Fixed